src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/view.html.twig

```diff
@@ -37,13 +37,13 @@
   </div>
 
   <div class="row">
-    <div class="col-md-4">
+    <div class="col-md-4" style="order: 2;">
       {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/customer.html.twig' %}
 
       {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/messages.html.twig' %}
     </div>
 
-    <div class="col-md-8">
+    <div class="col-md-12" style="order: 1;">
       {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/products.html.twig' %}
```

Comment:
this is temporary, the action of making the products block bigger must have been done in another pr